doc/manual/noteworthy-differences.rst

```diff
@@ -6,6 +6,39 @@
 Noteworthy Differences from other Languages
 *******************************************
 
+Noteworthy and unusual Julia features
+---------------------------------
+
+* Julia allows identifiers with complex Unicode, for example you could represent the value of a chi-squared test with ``χ²``,
```

Comment:
Great. Instead of replying to me, write that in documentation form, which is not what is here right now.